pkg/druid.go

```diff
@@ -726,6 +726,27 @@ func (ds *druidDatasource) prepareResponse(resp *druidResponse, settings map[str
 	response := backend.DataResponse{}
 	frame := data.NewFrame("response")
 	hideEmptyColumns, _ := settings["hideEmptyColumns"].(bool)
+	format, ok := settings["format"]
+	if !ok {
+		format = "long"
+	} else {
+		format = format.(string)
+	}
+	if format == "log" {
+		for ic, c := range resp.Columns {
+			var ff []string
+			ff = make([]string, 0)
+			if c.Type == "string" && c.Name == "message" {
```

Comment:
Do we need to accommodate scenarios for other types (i.e. metrics)?

Reply:
probably not since grafana treats the first string field it finds as the log message, so the message needs to be a string field. i am going to make the column name configurable, though.